cpp/README.md

```diff
@@ -4,7 +4,7 @@ gRPC CPP Wallet example
 Build the examples:
 
 ```
-bazel build :client :account-server :wallet-server :stats-server
+../tools/bazel build :client :account-server :wallet-server :stats-server
```

Comment:
The commands to run the binaries need update. The folder is `../bazel-bin/cpp/client`

Reply:
fixed